http.c

```diff
@@ -1024,6 +1024,11 @@ static int get_curl_http_version_opt(const char *version_string, long *opt)
 	return -1; /* not found */
 }
 
+static bool is_pkcs11_uri(const char *string)
+{
+	return string && strncasecmp(string, "pkcs11:", 7) == 0;
```

Comment:
On the Git mailing list, you'd most likely receive the feedback to use `istarts_with()` instead.